app/models/direct_file_data.rb

```diff
@@ -462,6 +462,11 @@ def fed_eic
     df_xml_value(__method__)&.to_i || 0
   end
 
+  def fed_eic=(value)
+    create_or_destroy_df_xml_node(__method__, true, 'WithholdingTaxAmt')
```

Comment:
[pebble] I'm seeing this node (`EarnedIncomeCreditAmt`) after `EstimatedTaxPaymentsAmt`, is that not right?(leslie_qss.xml) It's very possible I'm misunderstanding `create_or_destroy_df_xml_node`

<img width="750" alt="Screenshot 2024-11-06 at 9 51 37 AM" src="https://github.com/user-attachments/assets/2ba1c248-7ee0-4377-bcfd-ebbad52a1c2d">

Reply:
out of 48 personas there are about 28 that include the `EstimatedTaxPaymentsAmt` & `WithholdingTaxAmt` and 20 that only include `WithholdingTaxAmt`. I guess I could do something like but it after `EstimatedTaxPaymentsAmt` but if you can't find it put it after `WithholdingTaxAmt`